### ISSUE TITLE
Connect to socket.io server using options

### ISSUE THREAD

**Issue by optikfluffel** [netguru/ember-socket-guru#43]
💡 I want to connect to [RIPE Atlas Result Streams](https://atlas.ripe.net/docs/result-streaming/) and in their example they use the `path` option:

```js
// Create a connection (it can be also http on port 80)
var socket = io("https://atlas-stream.ripe.net:443", { path : "/stream/socket.io" });

// Subscribe to results coming from all the probes involved in the measurement 1791207
socket.emit("atlas_subscribe", { stream_type: "result", msm: 1791207 });

// Declare a callback to be executed when a measurement result is received
socket.on("atlas_result", function(result){
  console.log("I received ", result);
});
```

🤔 I can't figure out how to get `ember-socket-guru` to establish a connection. Simply using the full url as `host` doesn't seem to work and it keeps using `<host>/socket.io`. Is it somehow possible to pass down options to the underlying client libs?

**Comment by jbandura** [comment]
Thanks for pointing this out @optikfluffel! Unfortunately this was not possible :man_facepalming: I added a PR that fixes that: #46. Let me know if this fixed your issue!

**Comment by optikfluffel** [comment]
@jbandura after I installed `ember-lodash` manually it worked perfectly. Thanks ❤️

**Comment by jbandura** [comment]
I'm glad I was able to help :) Released in 1.0.1:
Released in 1.0.1 (https://github.com/netguru/ember-socket-guru/releases/tag/1.0.1), thanks again for reporting!
